eslint/babel-eslint-shared-fixtures/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@babel/eslint-shared-fixtures",
+  "version": "0.0.0",
+  "description": "Shared fixtures for testing @babel/eslint-* packages",
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
```

Comment:
Wanted to check to make sure this seemed like the best way to share these configs.

Reply:
We don't do anything like this currently, but I think that it's ok.